src/main/java/org/gridsuite/directory/server/repository/DirectoryElementEntity.java

```diff
@@ -49,6 +50,9 @@ public class DirectoryElementEntity {
     @Column(name = "description")
     private String description;
 
+    @Column(name = "creationDate")
+    private LocalDateTime creationDate;
```

Comment:
So, we can remove the 'creationDate' field, already stored in entities in actions-server, filter-server and study-server ...

Reply:
Agreed, here are the PRs : 
https://github.com/gridsuite/study-server/pull/303
https://github.com/gridsuite/actions-server/pull/59
https://github.com/gridsuite/filter-server/pull/49